plugins/portal-history-ornaments.js

```diff
@@ -0,0 +1,299 @@
+// @author         Johtaja
+// @name           Portal History&Ornaments
+// @category       Highlighter
+// @version        0.2.0
+// @description    Display portal history as highlighters, ornaments
+
+
+// use own namespace for plugin
+var portalsHistory = {};
+window.plugin.portalHighlighterPortalsHistory = portalsHistory;
```

Comment:
Is it meant to replace current `highlight-portal-history.js` or it is separate plugin?
If it's separate  then it should not take same namespace.

Also: `portal-history-ornaments.js` does not follow common convention about highlighters filenames.